src/org/myworldgis/netlogo/ProjectLatLon.java

```diff
@@ -0,0 +1,127 @@
+package org.myworldgis.netlogo;
+
+import org.nlogo.api.Argument;
+import org.nlogo.api.Context;
+import org.nlogo.api.ExtensionException;
+import org.nlogo.api.LogoException;
+import org.nlogo.api.LogoListBuilder;
+import org.nlogo.core.Syntax;
+import org.nlogo.core.SyntaxJ;
+import org.myworldgis.projection.Ellipsoid;
+import org.myworldgis.projection.Geographic;
+import org.myworldgis.projection.Projection;
+import org.ngs.ngunits.NonSI;
+import org.ngs.ngunits.SI;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.util.GeometryTransformer;
+
+
+/**
+ * 
+ */
```

Comment:
Unless I'm missing a reason for the empty block comments being here and in the rest of the file, they can be removed.